docs/tutorials/javascript/compliance/verify-credential.md

```diff
@@ -4,306 +4,115 @@ seo:
+- Have an XRP Ledger client library, such as **xrpl.js**, installed.
+- Know the issuer, subject, and credential type of the credential you want to verify. For purposes of this tutorial, you can use sample values of data that exists in the public network.
+    - For information on how to create your own credentials, see the [Build a Credential Issuing Service](../build-apps/credential-issuing-service.md) tutorial.
 
-This installs the appropriate version of the `xrpl.js` library and a few other dependencies. You can view all dependencies in the {% repo-link path="_code-samples/verify-credentials/js/package.json" %}`package.json`{% /repo-link %} file.
+## Source Code
 
-## Overview
+You can find the complete source code for this tutorial's examples in the {% repo-link path="_code-samples/verify-credential/" %}code samples section of this website's repository{% /repo-link %}.
 
-The Verify Credential sample code consists of one file, `verify_credential.js`, and contains two main parts:
+## Steps
 
-1. A function, `verifyCredential(...)` which can be called with appropriate arguments to verify that a credential exists and is valid. This function can be imported into other code to be used as part of a larger application.
-2. A commandline utility that can be used to verify any credential.
+### 1. Install dependencies
 
-## Usage
-
-To test that you have the code installed and working properly, you can run the commandline utility with no arguments to check the existence of a default credential on Devnet, such as:
+{% tabs %}
+{% tab label="JavaScript" %}
+From the code sample folder, use `npm` to install dependencies:
 
 ```sh
-node verify_credential.js
-```
-
-If all goes well, you should see output such as the following:
-
-```text
-info: Encoded credential_type as hex: 5465737443726564656E7469616C
-info: Looking up credential...
-info: {
-  "command": "ledger_entry",
-  "credential": {
-    "subject": "rBqPPjAW6ubfFdmwERgajvgP5LtM4iQSQG",
-    "issuer": "rPLY4DWhB4VA7PPZ8nvZLhShXeVZqeKif3",
-    "credential_type": "5465737443726564656E7469616C"
-  },
-  "ledger_index": "validated"
-}
-info: Found credential:
-info: {
-  "CredentialType": "5465737443726564656E7469616C",
-  "Flags": 65536,
-  "Issuer": "rPLY4DWhB4VA7PPZ8nvZLhShXeVZqeKif3",
-  "IssuerNode": "0",
-  "LedgerEntryType": "Credential",
-  "PreviousTxnID": "B078C70D17820069BDF913146F9908A209B4E10794857A3E474F4C9C5A35CA6A",
-  "PreviousTxnLgrSeq": 1768183,
-  "Subject": "rBqPPjAW6ubfFdmwERgajvgP5LtM4iQSQG",
-  "SubjectNode": "0",
-  "index": "F2ACB7292C4F4ACB18010251F1653934DC17F06AA5BDE7F484F65B5A648D70CB"
-}
-info: Credential is valid.
+npm i
 ```
+{% /tab %}
 
-If the code reports that the credential was not found when called with no arguments, it's possible that the example credential has been deleted or the Devnet has been reset. If you have another credential you can verify, you can provide the details as commandline arguments. For example:
+{% tab label="Python" %}
```

Comment:
We should remove/comment out the Python tab here.

Reply:
Commented out for now with a note to re-add when merging the tutorials